tls_codec/src/primitives.rs

```diff
@@ -7,6 +7,7 @@ use crate::{DeserializeBytes, SerializeBytes, U24};
 use super::{Deserialize, Error, Serialize, Size};
 
 use core::marker::PhantomData;
+use std::boxed::Box;
```

Comment:
Shouldn't this (and all the rest) also be guarded by the std feature?

Reply:
`Box` is available on `no_std` as `alloc::boxed::Box`